MyCompany.Seeq.Link.Connector.MyConnector/MyConnection.cs

```diff
@@ -154,11 +156,17 @@ public class MyConnection : ISignalPullDatasourceConnection {
             IEnumerator<DatasourceSimulator.Tag> tags = this.datasourceSimulator.Tags;
             while (tags.MoveNext()) {
                 DatasourceSimulator.Tag tag = tags.Current;
+                string tagId = string.Format("{0}", tag.Id);
+
+                this.createChildAsset(rootAssetId, tagId, tag.Name);
+
                 SignalWithIdInputV1 signal = new SignalWithIdInputV1();
```

Comment:
I think it would be good to extract the creation of Signal from a tag into a separate method.